Marlin/ubl_G29.cpp

```diff
@@ -1367,20 +1369,24 @@
           if (far_flag) {
             for (uint8_t k = 0; k < GRID_MAX_POINTS_X; k++) {
               for (uint8_t l = 0; l < GRID_MAX_POINTS_Y; l++) {
-                if (!isnan(ubl.z_values[k][l])) {
-                  distance += sq(i - k) * (MESH_X_DIST) * .05
-                            + sq(j - l) * (MESH_Y_DIST) * .05;
+                if (i != k && j != l && !isnan(ubl.z_values[k][l])) {
```

Comment:
That code has been changing a lot...   I'm trying to find a good algorithm that tries to select the next point that is far away from any other point.    It should not be as difficult as it is turning out to be.    The intent is to get a bunch of points all spread out on the bed (until the user says to stop) because that helps the physical bed leveling.   

So far...   I don't have a good algorithm for doing that....